### DISCUSSION TITLE
Git flow

### DISCUSSION THREAD

**Discussion by IhorNehrutsa** [CAP1Sup/Intellistep#70] [Ideas]
Maybe it will be convenient to make releases

git-flow cheatsheet
https://danielkummer.github.io/git-flow-cheatsheet/index.html

A successful Git branching model
https://nvie.com/posts/a-successful-git-branching-model/

**Comment by CAP1Sup** [comment]
I like the idea of using the method in the second link. I'll add it to the documentation and start abiding by it as best I can.

**Comment by IhorNehrutsa** [comment]
The first link shows how to use a software implementation of the idea from the second link: install, init, use. 
The main advantage is simple synchronisation of branches.

**Reply by CAP1Sup** [reply]
How do you think the implementation is going so far? I've tried to stick with the guide that you provided. I'd prefer not to use git-flow just because it requires Linux (or a messy Windows/Linux combination)

**Comment by IhorNehrutsa** [comment]
There is no dogma in code development. 
All ways are possible.

Maybe we return later.
Keep it in mind.
